MAUI/Picker/getting-started.md

```diff
@@ -0,0 +1,308 @@
+
+In SfPicker control, validation buttons (OK and Cancel) can be customized by setting the [OkButtonText] and [CancelButtonText] property in the [PickerFooterView]. It allows you to confirm or cancel the selected item.  The `OkButtonText` can be enabled by using the [ShowOkButton] property in [PickerFooterView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3" %}
+
+<picker:SfPicker x:Name="Picker">
+    <picker:SfPicker.FooterView>
+        <picker:PickerFooterView ShowOkButton="True" Height="40" />
+    </picker:SfPicker.FooterView>
+</picker:SfPicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="4 5" %}
+
+SfPicker picker = new SfPicker();
+picker.FooterView= new PickerFooterView()
+{  
+    ShowOkButton = true,
+    Height = 40,
+};
+
+this.Content = picker;
+
+{% endhighlight %}  
+{% endtabs %}
+
+## Set column header to the Picker items
+
+The SfPicker control allows you to the add the column header text by setting the [HeaderText] property in [PickerColumn] and you need to add the column header height by setting the [Height] property in [PickerColumnHeaderView]. Its text style and background color are customizable.
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="7 12" %}
+
+<picker:SfPicker x:Name="picker">
+<picker:SfPicker.HeaderView>
+    <picker:PickerHeaderView Text="Select a color" Height="40" />
+</picker:SfPicker.HeaderView>
+
+<picker:SfPicker.Columns>
+    <picker:PickerColumn HeaderText="Colors" 
+                         ItemsSource="{Binding DataSource}" />
+</picker:SfPicker.Columns>
+
+<picker:SfPicker.ColumnHeaderView>
+    <picker:PickerColumnHeaderView Height="40"
+                                   Background="#6750A4">
+        <picker:PickerColumnHeaderView.TextStyle>
+            <picker:PickerTextStyle TextColor="White" FontSize="15" />
+        </picker:PickerColumnHeaderView.TextStyle>
+    </picker:PickerColumnHeaderView>
+</picker:SfPicker.ColumnHeaderView>
+</picker:SfPicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="14 21" %}
+
+ItemInfo itemInfo = new ItemInfo();
+SfPicker picker = new SfPicker()
+{
+    HeaderView = new PickerHeaderView()
+    {
+        Text = "Select a color",
+        Height = 50,
+    },
+
+    Columns = new ObservableCollection<PickerColumn>()
+    {
+        new PickerColumn()
+        {
+            HeaderText = "Colors",
+            ItemsSource = itemInfo.DataSource,
+        }
+    },
+
+    ColumnHeaderView = new PickerColumnHeaderView()
+    {
+        Height = 40,
+        Background = Color.FromRgba("#6750A4"),
+        TextStyle = new PickerTextStyle()
+        {
+            TextColor = Colors.White,
+            FontSize = 15,
+        },
+    },
+};
+
+this.Content = picker;
+
+{% endhighlight %}  
+{% endtabs %}
+
+## Set height and width to the Picker
+
+The SfPicker control allows you to change the height and width by using the [HeightRequest] and [WidthRequest] property in [SfPicker].
```

Comment:
The SfPicker control allows you to change the height and width by using the [HeightRequest] and [WidthRequest] properties in the [SfPicker].

Reply:
Content modified.